src/App.js

```diff
@@ -24,9 +24,9 @@ import { Lobby } from 'boardgame.io/react';
 //   )
 // }
 
-const server = `https://${window.location.hostname}`;
+// const server = `https://${window.location.hostname}`;
 // development mode
-// const server = http://${window.location.hostname}:8000
+const server =  `http://${window.location.hostname}:8000`;
```

Comment:
can you revert this change? that way the code is good to be deployed as-is.

Reply:
Ah, sorry about that! Just fixed in latest commits.